frameworks/kafka/tests/test_upgrade.py

```diff
@@ -28,58 +19,23 @@ def teardown_module(module):
     install.uninstall(SERVICE_NAME)
 
 
+@pytest.mark.skip(reason="INFINITY-1734")
+@pytest.mark.upgrade
```

Comment:
There should be a couple PRs that can go in today, that will allow this to be enabled.

In particular this PR and this line, re-enables this test: https://github.com/mesosphere/dcos-commons/pull/1122/files#diff-2640c88535e40fe119dc4850bd2638d2L33